resolver/Cargo.toml

```diff
@@ -46,13 +44,11 @@ dnssec = []
 name = "trust_dns_resolver"
 path = "src/lib.rs"
 
-[build-dependencies]
-lalrpop = "^0.13.1"
-
 [dependencies]
+# resolv-conf = { git = "https://github.com/tailhook/resolv-conf" }
+resolv-conf = { path = "../../resolv-conf/" }
```

Comment:
that won't compile due to this. I'll uncomment the line above one when https://github.com/tailhook/resolv-conf/pull/5 is merged.